plugins/jobs/setup.py

```diff
@@ -0,0 +1,10 @@
+from setuptools import setup
+
+
+setup(
+    name='girder-plugin-jobs',
+    py_modules=['girder_plugin_jobs'],
+    entry_points={
+        'girder.plugin': 'jobs = girder_plugin_jobs:load'
+    }
+)
```

Comment:
This is a minimal `setup.py` (actually the `py_modules` might not be necessary).  The key point that makes it a girder plugin is the `entry_points` line.  The entry point name is `girder.plugin` and the value string means the plugin name is `jobs` and the entrypoint function is in the module `girder_plugin_jobs` called `load`.  The naming is arbitrary, plugins can point the entry point to any function.